sympy/matrices/tests/test_normalforms.py

```diff
@@ -1,23 +1,31 @@
 from sympy import Symbol, Poly
-from sympy.polys.solvers import RawMatrix as Matrix
+from sympy.polys.solvers import RawMatrix
```

Comment:
We could probably ignore this. RawMatrix has been essentially superseded by PolyMatrix that is more flexible.